testing/mock-server/server/server.go

```diff
@@ -314,7 +314,7 @@ func FindStub(service, method string, in, out interface{}) error {
 
 	if resp.StatusCode != http.StatusOK {
 		body, _ := io.ReadAll(resp.Body)
-		return fmt.Errorf(string(body))
+		return fmt.Errorf("%s", string(body))
```

Comment:
If you do not need to format error message, you can use `errors.New`

Reply:
good addition - changes made to address this particular issue.